src/main/target/common_post.h

```diff
@@ -25,3 +25,11 @@
 # undef VTX_SMARTAUDIO
 # undef VTX_TRAMP
 #endif
+
+#if defined(SPEKTRUM_BIND_PIN) && !defined(SPEKTRUM_BIND)
+#define SPEKTRUM_BIND
+#endif
+
+#if !defined(SPEKTRUM_BIND_PIN) && defined(SPEKTRUM_BIND)
```

Comment:
Wouldn't it be better just to change all occurrences of `#ifdef SPEKTRUM_BIND` to `#ifdef SPECTRUM_BIND_PIN`, so that there is only one `#define`?